build.gradle

```diff
@@ -92,7 +92,7 @@ springBoot {
 }
 
 tasks.withType(JavaCompile).each {
-    it.options.compilerArgs.add('--enable-preview')
+//    it.options.compilerArgs.add('--enable-preview')
```

Comment:
Should this be part of the PR or is it just to avoid the failing build task?

Reply:
just to avoid the failing build task

Reply:
I just removed it, because Stephan Krusche provided a solution for it